integration-tests/testconfig/default.toml

```diff
@@ -13,10 +13,9 @@ log_producer_retry_limit = 10
 [ChainlinkImage]
 # postgres version to use
 postgres_version = "15.6"
-# chainlink image to use
-image = "public.ecr.aws/chainlink/chainlink"
 # chainlink image tag to use
 version = "2.12.0"
```

Comment:
@lukaszcl do we need this version here? so I see that I can specify CL node version by `chainlink_version` in GHA workflow and it is required. But I would make it optional and if its not specified, then use latest develop version

Reply:
@iljapavlovs I did not change the line with `version`. I'd say it's better to keep it there for visibility. It will be overriden anyway by either config override or `E2E_TEST_CHAINLINK_VERSION` env.